packages/wallets/src/components/AccountsList/AccountsList.tsx

```diff
@@ -19,12 +20,12 @@ const AccountsList = ({ isWalletSettled }: TProps) => {
     const { isMobile } = useDevice();
     const [isMT5PlatformListLoaded, setIsMT5PlatformListLoaded] = useState(false);
     const [isOptionsAndMultipliersLoaded, setIsOptionsAndMultipliersLoaded] = useState(false);
+    const { t } = useTranslation();
 
     if (isMobile) {
         return (
             <WalletsPrimaryTabs className='wallets-accounts-list'>
-                {/* TODO: Localization needed on tab headers */}
-                <WalletsPrimaryTabList list={['CFDs', 'Options & multipliers']} />
+                <WalletsPrimaryTabList list={[t('CFDs'), t('Options & multipliers')]} />
                 <WalletsPrimaryTabPanels>
                     <WalletsPrimaryTabPanel>
                         <CFDPlatformsList onMT5PlatformListLoaded={setIsMT5PlatformListLoaded} />
```

Comment:
on mobile, I'm able to remove CFDPlatformsList and OptionsAndMultipliersListing without tests detecting it,

Reply:
Updated and fixed